docs/hardware/_sidebar.md

```diff
@@ -0,0 +1,2 @@
+- [Binos Outdoor Air Monitor](hardware/binos_outdoor_air_monitor.md)
+- [AirQo Outdoor Monitor](hardware/airQo_outdoor_monitor.md)
```

Comment:
![Screenshot from 2022-11-07 18-51-09](https://user-images.githubusercontent.com/65954740/200354338-c77cce26-2747-4ed3-a33b-a5002377fc72.png)
![Screenshot from 2022-11-07 18-51-43](https://user-images.githubusercontent.com/65954740/200354406-ac2a4c87-50dd-49b9-8189-c353202c2b19.png)

@123MwanjeMike the [AirQo Monitor guide](https://drive.google.com/file/d/1LotNVXHDYedjNEN_V8QQA6bY-i-CAE5-/view?usp=sharing) shows there are 2 guides within the hardware guide, but the content seemed like it for one file...please guide on how to go about this because one of the two side bar items (AirQo Outdoor Monitor and Binos Outdoor Air Monitor) turns out to be empty

Reply:
Thanks @HassanBahati  for catching this.
@NsubugaAnold / @deogratius2phantom , could you provide some guidance on this?